### PR TITLE
Fix reference context for clear sRGB target

### PR DESCRIPTION
**Pull request by qkmiao**

### CODE REVIEW

File: sdk/tests/deqp/framework/opengl/simplereference/sglrReferenceContext.js [hunk 1]
```diff
@@ -3133,7 +3133,8 @@ goog.scope(function() {
         if (hasColor0 && (buffers & gl.COLOR_BUFFER_BIT) != 0) {
             /** @type {Array<number>} */ var colorArea = deMath.intersect(baseArea, sglrReferenceContext.getBufferRect(colorBuf0));
             access = colorBuf0.getSubregion(colorArea);
-            /** @type {Array<number>} */ var c = this.m_clearColor;
+            /** @type {boolean} */ var isSRGB = colorBuf0.raw().getFormat().isSRGB();
+            /** @type {Array<number>} */ var c = (isSRGB && this.m_sRGBUpdateEnabled) ? tcuTextureUtil.linearToSRGB(this.m_clearColor) : this.m_clearColor;
```
Comment (qkmiao) [inline sdk/tests/deqp/framework/opengl/simplereference/sglrReferenceContext.js]: C++ code:
https://android.googlesource.com/platform/external/deqp/+/deqp-dev/framework/opengl/simplereference/sglrReferenceContext.cpp#3374

**Comment by Kangz** [comment]
LGTM, thank you!

**Comment by zhenyao** [comment]
One question: does this bug cause some tests to fail?

**Comment by qkmiao** [comment]
No tests failed because of this bug.  There is no test on sRGB color clear in dEQP. I found this when I debugging framebufferblit failures on Intel Mesa driver.

**Pull request merged**